dotCMS/build.gradle

```diff
@@ -366,33 +360,19 @@ dowloadStarterForTests.description = 'Downloads the test starter zip to the inte
 
 ext.prepareFelixExternalFolder = { String felixExternalFolder ->
 
-    println "Configuring felix in folder [" + felixExternalFolder + "]"
 
     if (felixExternalFolder.startsWith("/")) {// We need an absolute path here
-
         copy {
-            from configurations.felix
+            from felixBundleSourceFolder
             into felixExternalFolder.concat("/bundle")
+            include '**/*.jar'
         }
 
         copy {
             from 'src/main/webapp/WEB-INF/felix/osgi-extra-generated.conf'
             into felixExternalFolder
         }
 
-
-        copy {
```

Comment:
Will we keep using the http felix dependency? I had removed it as you advised but the tika plugin didn't deploy. I had to add it back. The updated tika jar with the changes suggested was already uploaded to artifactory